config/locales/liquid_tags/en.yml

```diff
@@ -47,6 +47,7 @@ en:
     liquid_tag_base:
       invalid_context: Invalid context. This liquid tag can only be used in %{valid}.
       no_source_found: No source found
+      no_embed_word: Check your embed tag
```

Comment:
there's a file fr.yml that's in the same directory, any key (like `no_embed_word`) added to the English file should go in the French file as well.

As a general rule, there's no need to know French when you do that (it's okay to include the same string "Check your embed tag" there, and a translator will update it from the English when ready).